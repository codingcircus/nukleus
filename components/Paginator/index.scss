@import '~kununu-theme/scss/base/variables';

@media (max-width: $screen-xs-min) {
  .paginator ul > li > a {
    padding: 11px;
  }
}
