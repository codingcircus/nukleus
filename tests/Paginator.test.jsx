import React from 'react';
import renderer from 'react-test-renderer';
import Paginator from 'Paginator'; // eslint-disable-line import/no-unresolved, import/extensions, import/no-extraneous-dependencies

test('Renders Paginator without crashing', () => {
  const component = renderer.create(
    <Paginator
      totalPages={10}
      pathname="2" />
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
