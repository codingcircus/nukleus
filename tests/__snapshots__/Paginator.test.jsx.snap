exports[`test Renders Paginator without crashing 1`] = `
<div
  className="paginator">
  <ul
    aria-label="Pagination"
    className="pagination"
    role="navigation">
    <li
      className="disabled pointer-disabled">
      <a
        onClick={[Function]}
        style={Object {}}>
        <
      </a>
    </li>
    <li
      className="active pointer-disabled">
      <a
        onClick={[Function]}
        style={Object {}}>
        1
      </a>
    </li>
    <li
      className="inActive">
      <a
        onClick={[Function]}
        style={Object {}}>
        2
      </a>
    </li>
    <li
      className="inActive">
      <a
        onClick={[Function]}
        style={Object {}}>
        3
      </a>
    </li>
    <li
      className="inActive">
      <a
        onClick={[Function]}
        style={Object {}}>
        4
      </a>
    </li>
    <li
      className="inActive">
      <a
        onClick={[Function]}
        style={Object {}}>
        5
      </a>
    </li>
    <li
      className="inActive">
      <a
        onClick={[Function]}
        style={Object {}}>
        6
      </a>
    </li>
    <li
      className="inActive">
      <a
        onClick={[Function]}
        style={Object {}}>
        7
      </a>
    </li>
    <li
      className={false}>
      <a
        onClick={[Function]}
        style={Object {}}>
        >
      </a>
    </li>
  </ul>
</div>
`;
